app/services/weetsService.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const api = axios.create({
+  baseURL: 'http://numbersapi.com',
+  timeout: 3000
+});
+
+const getWeet = async () => {
+  try {
+    const { data } = await api.get('/random');
+    return data;
+  } catch (error) {
+    throw error;
```

Comment:
Create and error using the errors middelware and the errors.js file, check it please. If you have doubts let me know.